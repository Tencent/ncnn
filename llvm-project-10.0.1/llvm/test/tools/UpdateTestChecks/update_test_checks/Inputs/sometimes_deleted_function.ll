; RUN: opt -S < %s | FileCheck %s --check-prefixes=ALL,FIRST
; RUN: opt -S -globalopt < %s | FileCheck %s --check-prefixes=ALL,SECOND
;
; Make sure we use FIRST to check for @sometimes_here as ALL does not work.

define internal void @sometimes_here() {
  ret void
}

define void @always_here() {
  ret void
}
