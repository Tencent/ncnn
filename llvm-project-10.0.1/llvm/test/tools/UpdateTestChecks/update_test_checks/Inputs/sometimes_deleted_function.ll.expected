; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
; RUN: opt -S < %s | FileCheck %s --check-prefixes=ALL,FIRST
; RUN: opt -S -globalopt < %s | FileCheck %s --check-prefixes=ALL,SECOND
;
; Make sure we use FIRST to check for @sometimes_here as ALL does not work.

define internal void @sometimes_here() {
; FIRST-LABEL: @sometimes_here(
; FIRST-NEXT:    ret void
;
  ret void
}

define void @always_here() {
; ALL-LABEL: @always_here(
; ALL-NEXT:    ret void
;
  ret void
}
